project/Dependencies.scala

```diff
@@ -88,6 +88,8 @@ object Dependencies {
     "org.mongodb" %% "casbah" % "3.0.0"
   )
 
+  val sparkBackendDependencies = List("org.json4s" %% "json4s-jackson" % "3.4.0")
```

Comment:
1) It is easier to use than Spray Json
2) Some boilerplate code needs to be included if change to Spray Json
3) Doesn't work in a straight forward way if case class has Date or some non-primitive data types.

However here we are using primitive data types, it would not matter.